test/e2e/allocator_test.go

```diff
@@ -223,7 +223,7 @@ func createAllocationPolicy(t *testing.T, p *multiclusterv1alpha1.GameServerAllo
 	t.Logf("created allocation policy %v", policy)
 }
 
-func getAllocatorEndpoint(t *testing.T) (string, int32) {
+func getAllocatorEndpoint(t *testing.T) (ip string, intPort int32) {
```

Comment:
Is this left over from the old check?

Reply:
yes, did not notice these 2 named results. Fixed now